dor/service_layer/message_bus/memory_message_bus.py

```diff
@@ -13,21 +14,21 @@ def register_event_handler(self, event_type: Type[Event], handler: Callable):
             self.event_handlers[event_type] = []
         self.event_handlers[event_type].append(handler)
 
-    def handle(self, message, uow: UnitOfWork = None):
+    def handle(self, message, uow: UnitOfWork):
         # Handles a message, which must be an event.
         if isinstance(message, Event):
             self._handle_event(message, uow)
         else:
             raise ValueError(f"Message of type {type(message)} is not a valid Event")
 
-    def _handle_event(self, event: Event, uow: UnitOfWork = None):
+    def _handle_event(self, event: Event, uow: UnitOfWork):
         # Handles an event by executing its registered handlers.
         if event.__class__ in self.event_handlers:
             queue = [event]
             while queue:
                 next_event = queue.pop(0)
                 for handler in self.event_handlers[type(next_event)]:
-                    handler(next_event, uow)
+                    handler(next_event)
```

Comment:
This change is an alternate fix to adding the `uow` to the lambda signature. Looking at Chapter 13 of APwP, I think this is more in line with what they suggest: https://learning.oreilly.com/library/view/architecture-patterns-with/9781492052197/ch13.html#idm45846291766760